packages/evm/src/opcodes/functions.ts

```diff
@@ -61,16 +62,40 @@ export interface AsyncOpHandler {
 
 export type OpHandler = SyncOpHandler | AsyncOpHandler
 
+// TODO: verify that this is the correct designator
+// The PR https://github.com/ethereum/EIPs/pull/8969 has two definitions of the
+// designator: the original (0xef0100) and the designator added in the changes (0xef01)
+const eip7702Designator = hexToBytes('0xef01')
```

Comment:
Just confirming my thinking.  These only get computed once at run time, right?  They aren't recomputed every single time we access this code are they?

Reply:
Only computed once! Can also move it somewhere else like `types.ts`.